SayTheirNames/Source/Controller/Home/HomeView.swift

```diff
@@ -82,26 +138,10 @@ final class HomeView: UIView {
         return separator
     }()
     
-    func safeWidth(for collectionView: UICollectionView) -> CGFloat {
-        let width = collectionView.frame.width -
-            collectionView.safeAreaInsets.left -
-            collectionView.safeAreaInsets.right -
-            collectionView.layoutMargins.left -
-            collectionView.layoutMargins.right
-        
-        var flowLayoutMargins: CGFloat = 0
-        
-        if let flowLayout = collectionView.collectionViewLayout as? UICollectionViewFlowLayout {
-            flowLayoutMargins = flowLayout.sectionInset.left + flowLayout.sectionInset.right
-        }
-        
-        return width - flowLayoutMargins
-    }
-    
     override func didMoveToSuperview() {
         super.didMoveToSuperview()
         createLayout()
-        backgroundColor = .black
+        backgroundColor = .black // needed?
```

Comment:
I believe this is a simple work around to handle the custom nav bar not handling the status bar area, which would show as white if this wasn't being set